app/config/api.yml

```diff
@@ -3,7 +3,7 @@ sylius_resource:
         acme.post:
             driver: doctrine/orm
             classes:
-                model: AppBundle\Entity\Post
+                model: DemoBundle\Entity\Post
```

Comment:
You would need to import it anyway. I'd keep it here. You will want to add resources from AppBundle.